include/uxr/agent/transport/udp/UDPServerWindows.hpp

```diff
@@ -29,11 +29,11 @@ namespace uxr {
 class UDPServer : public UDPServerBase
 {
 public:
-    microxrcedds_agent_DllAPI UDPServer(uint16_t port);
+    microxrcedds_agent_DllAPI UDPServer(uint16_t port, uint16_t discovery_port);
     microxrcedds_agent_DllAPI ~UDPServer() = default;
 
 private:
-    virtual bool init() override;
+    virtual bool init(bool discovery_enabled) override;
```

Comment:
To keep consistency remove `virtual` from derived class overridden methods. If the derived class should not have more derived, consider `final` keyword.

Inconsistency between SerialServers and TCP and UDP servers.